sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1208,6 +1208,14 @@ class Lasso(ElasticNet):
     If so, then additionally check whether the dual gap is smaller than `tol` times
     :math:`||y||_2^2 / n_{\text{samples}}`.
 
+    The target can be a 2-dimensional array, resulting in the optimization of the
+    following objective:
+    :math:`(1 / (2 * n_{samples})) * ||Y - XW||^2_F + \alpha * ||W||_{1,1}`, where
```

Comment:
The rendering shows a "Math input error". We should check if we did not make a mistake here. I think we can remove the `*` for multiply

Reply:
@glemaitre I was missing a double backslash for `\\alpha`. I actually removed the `math` environment to stay consistent with the format used at the top of the docstring where the Lasso objective is presented.